compressor/tests/test_offline.py

```diff
@@ -62,8 +62,44 @@ def setUp(self):
             'COMPRESS_OFFLINE': True
         }
 
+        if django.VERSION >= (1, 8):
+            override_settings['TEMPLATES'] = [
+                {
+                    "BACKEND": "django.template.backends.django.DjangoTemplates",
+                    "APP_DIRS": True,
+                    "DIRS": override_settings['TEMPLATE_DIRS'],
+                },
+            ]
+
         if "jinja2" in self.engines:
-            override_settings["COMPRESS_JINJA2_GET_ENVIRONMENT"] = lambda: self._get_jinja2_env()
+            if django.VERSION < (1, 8):
+                override_settings["COMPRESS_JINJA2_GET_ENVIRONMENT"] = lambda: self._get_jinja2_env()
+            else:
+                override_settings['TEMPLATES'] += [
```

Comment:
Cuz in django 1.8, the `TEMPLATES` is a settings entry which must pass in a `dict`, and the `override_settings` must pass a root entry in. So that's why I have to repeat them again.

I mean, theses lines are used to override the `TEMPLATE_DIRS` populated at line 60, right? But I could not write such `override_settings['TEMPLATES'][0]['DIRS'] = override_settings['TEMPLATE_DIRS']`. The `override_settings` doesn't accept this.